src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -0,0 +1,150 @@
+        resource = u"/highriskemployee/add"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def set_alerts_enabled(self, enabled=True):
+        data = {
+            "tenantId": self._user_context.get_current_tenant_id(),
+            "alertsEnabled": enabled,
+        }
+        resource = u"/highriskemployee/setalertstate"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def remove(self, user_id):
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/highriskemployee/remove"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def get(self, user_id):
+        data = {"tenantId": self._user_context.get_current_tenant_id(), "userId": user_id}
+        resource = u"/user/getbyid"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+    def search(
+        self,
+        risk_tags,
+        filter_type="OPEN",
+        sort_key=None,
+        sort_direction=None,
+        page_size=100,
+        page_num=1,
+    ):
+        """
+        Search High Risk employee list. Filter results by filter_type or risk factors.
+
+        #TODO Param description
+        """
+
+        # Defined default values for page_size & page_num even though in requirement
+        # it was not mentioned. Did not see any validation in other existing modules
+        # hence didn't add them.
+
+        data = {
+            "tenantId": self._user_context.get_current_tenant_id(),
+            "filterType": filter_type,
+            "riskFactors": risk_tags,
+            "pgSize": page_size,
+            "pgNum": page_num,
+        }
+        if sort_key:
+            data["srtKey"] = sort_key
+        if sort_direction:
+            data["srtDirection"] = sort_direction
+
+        resource = u"/highriskemployee/search"
+        uri = u"{0}{1}".format(self._uri_prefix, resource)
+        return self._session.post(uri, data=json.dumps(data))
+
+
+class DetectionListUserClient(BaseClient):
+    """
+    TODO Documentation here
+    """
+
+    _api_version = u"v2"
+    _uri_prefix = u"/svc/api/{0}/user".format(_api_version)
+
+    def __init__(self, session, user_context):
+        super(DetectionListUserClient, self).__init__(session)
+        self._user_context = user_context
+
+    def update_notes(self, user_id, notes):
+
+        data = {
+            "tenantId": self._user_context.get_current_tenant_id(),
+            "userId": user_id,
+            "notes": notes,
+        }
+        uri = u"{0}/{1}".format(self._uri_prefix, u"updatenotes")
+        return self._session.post(uri, data=json.dumps(data))
+
+    def add_risk_tag(self, user_id, tags=None):
```

Comment:
tags should not be an optional param here

Reply:
Done